### DISCUSSION TITLE
vue3 引入x6 控制台报错 TypeError: Cannot read properties of undefined (reading 'ToolItem')     at button.ts

### DISCUSSION THREAD

**Discussion by Epic-Deno** [antvis/X6#3350] [Q&A]
<img width="852" alt="image" src="https://user-images.githubusercontent.com/74635349/223134125-731de646-526f-4bfd-a6f7-782138c8a610.png">

环境版本： 
![Uploading image.png…]()

**Comment by NewByVector** [comment]
参考这个 https://github.com/antvis/X6/issues/1442

**Reply by Epic-Deno** [reply]
我以及解决了， 是vite版本的问题， vite:2.9.8有问题， 升级到vite3.0.0就没问题了
